carrr-client/src/views/administration/NewCategoryRule.vue

```diff
@@ -0,0 +1,458 @@
+<template>
```

Comment:
Admit you just made it this big to collect number of lines!

Reply:
How dare you?!?